### DISCUSSION TITLE
Welcome to trivy-operator Discussions!

### DISCUSSION THREAD

**Discussion by chen-keinan** [aquasecurity/trivy-operator#45] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by r0binak** [comment]
What is the essential difference between starboard operator and trivy operator?

**Reply by chen-keinan** [reply]
Starboard is a kubernetes scanner which  has been built for pluggable scanners (trivy, contest, polaris etc.).

Trivy-operator will essentially be getting the trivy (only scanner) capabilities on kubernetes with continued scanning. 
(Operator).
The idea is to introduce the kubernetes capabilities to Trivy CLI and use the same Trivy CLI with Operator. so it makes the Operator Trivy Native.
Adding links to docs: 
[Trivy K8s CLI](https://aquasecurity.github.io/trivy/latest/docs/kubernetes/cli/scanning/)
 [Trivy K8s Operator](https://aquasecurity.github.io/trivy/latest/docs/kubernetes/operator/)
